### PR TITLE
Upgrade go version to 1.14.4

### PR DESCRIPTION
**Pull request by RA489**
Upgrade go version to 1.14.4

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,7 +29,7 @@ DEB_VERSION ?= $(subst -,~,$(RAW_VERSION))
 RPM_VERSION ?= $(DEB_VERSION)
 
 # used by hack/jenkins/release_build_and_upload.sh and KVM_BUILD_IMAGE, see also BUILD_IMAGE below
-GO_VERSION ?= 1.13.9
+GO_VERSION ?= 1.14.4
```
Comment (afbjorklund) [inline Makefile:32]: Need to update the BUILD_IMAGE below, and also the link:

```diff
- # https://github.com/kubernetes/kubernetes/blob/release-1.18/build/build-image/cross/VERSION
- BUILD_IMAGE 	?= us.gcr.io/k8s-artifacts-prod/build-image/kube-cross:v$(GO_VERSION)-5
+ https://github.com/kubernetes/kubernetes/blob/master/build/build-image/cross/VERSION
+ BUILD_IMAGE 	?= us.gcr.io/k8s-artifacts-prod/build-image/kube-cross:v$(GO_VERSION)-2
```

Comment (RA489) [reply Makefile:32]: @afbjorklund updated.

**Pull request merged**